calibration/DOSTAD/CGINS-DOSTAD-00422__20141012.csv

```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+422,CC_conc_coef,"[0.0, 1.0]",default
+422,CC_csv,"[0.00280486, 0.000117958, 2.451116e-06, 229.393, -0.397540, -59.2715, 4.54200]",
```

Comment:
3rd coefficient should be 2.45111e-06 (ther is an extra 6 at the end before the sci note)
NOT ok by JL & CD